src/js/fc.js

```diff
@@ -109,8 +109,15 @@ var FC = {
         };
         
         MIXER_CONFIG = {
-            mixer:                      0,
+            mixer:                      false,
             reverseMotorDir:            0,
+            load: function(callback) {
```

Comment:
This will cause problems with backup / restore, where the objects in `fc.js` are serialised / deserialised. Please change this to do an explicit `MSP.send_message(MSPCodes.MSP_MIXER_CONFIG, false, false, callback)` in the tabs where the mixer config is needed.

Reply:
I've tried to find all the places where this new `load()` function shall be called, it seem it even ended up be called twice at one place. I removed the duplicate and double checked references to `MIXER_CONFIG` and it should be okay now (I'm referring to code not yet pushed). Would this particular object be okay like this (fixing the serialization problem):
```
        MIXER_CONFIG = {
            mixer:                      0,
            reverseMotorDir:            0,
            _loaded:                    false,
            load: function(callback) {
                if (MIXER_CONFIG._loaded === false) {
                    MSP.send_message(MSPCodes.MSP_MIXER_CONFIG, false, false, function() {
                        MIXER_CONFIG._loaded = true;
                        callback();
                    });
                } else {
                    callback();
                }
            },
        }; 
```